requirements-ext.txt

```diff
@@ -4,6 +4,7 @@ pytest>=2.3
 flake8>=2.1.0
 pycparser>=2.10
 mpi4py>=1.3.1
+py-cpuinfo
```

Comment:
Slightly concerned about adding this package as maintenance seems to have ceased. This could be problematic moving to the M1 Mac (https://github.com/workhorsy/py-cpuinfo/issues/162), but I might be wrong. I will try and look into an alternative and report back

Reply:
Aii, sorry. I had added this dependency two years ago and didn't look into it again. Thanks for trying to find an alternative, Jack!

Reply:
Doesn't look like there are great alternatives. I wouldn't mind, querying `platform` and hard-coding its [parameters](https://discussions.apple.com/thread/252073619).

Reply:
I'm 100% in favour of just hardcoding `PYOP2_SIMD_WIDTH=4` but for my alternative implementation of what is already implemented here see https://github.com/OP2/PyOP2/pull/665